test/pytest/test_peer.py

```diff
@@ -71,7 +71,7 @@ async def test_init_failure(rule, init_type, monkeypatch, transport):
 
 
 @pytest.mark.asyncio
-async def test_immediate_shutdown(rule):
```

Comment:
...can't we just drop `async`?

Reply:
```
rule = <cockpit.peer.PeerRoutingRule object at 0x72ad944847d0>

    @pytest.mark.asyncio
    def test_immediate_shutdown(rule):
>       peer = rule.apply_rule({'payload': 'test'})

test/pytest/test_peer.py:75:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src/cockpit/peer.py:275: in apply_rule
    self.peer = ConfiguredPeer(self.router, self.config)
src/cockpit/peer.py:245: in __init__
    super().__init__(router)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <cockpit.peer.ConfiguredPeer object at 0x72ad94914810>, router = <test.pytest.test_peer.Bridge object at 0x72ad94487a10>

    def __init__(self, router: Router):
        super().__init__(router)

        # All Peers start out frozen — we only unfreeze after we see the first 'init' message
        self.freeze_endpoint()

>       self.init_future = asyncio.get_running_loop().create_future()
E       RuntimeError: no running event loop

src/cockpit/peer.py:51: RuntimeError
```

Reply:
ideally, Peer wouldn't have this dependency...